igcollect/linux_disklatency.py

```diff
@@ -107,5 +81,50 @@ def main():
             print(template.format(mode, k, int(v)))
 
 
+def get_path(args) -> str:
+    if args.path is not None:
+        return args.path
+
+    # no --path is specified -> use libvirt to get the storage volume
+    from libvirt import (
+        open as libvirt_open,
+        libvirtError,
+    )
```

Comment:
If I understood it correctly from before, it should not be clear and mandatory, that libvirt is available.
That would mean you would want to try and import those modules here:
``` python
try:
    from libvirt import ...
    # do libvirt stuff
except ImportError:
    # do alternative stuff

Reply:
if no --path is set, "libvirt" is mandatory as it will extract the storage volumes based on the other parameters. Therefore no try/expect around it to get the plain exception in case (as before) .

Reply:
I see, then nevermind. You could try and check if the import fits in one line actually. Looks to me like it could. ^^